src/hooks/use-collapsible/use-collapsible.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const useCollapsible = (defaultValue = true) => {
```

Comment:
I’m wondering if we should name it like this or just something like `useToggleState`, because the hook itself does not do any “collapsible” work... 🤔

Reply:
yeah that makes sense.

Reply:
[here](https://github.com/commercetools/ui-kit/pull/966/commits/5fb91aef6008a84dfea79a779f3539737d6dbeb7)